### ISSUE TITLE
TypeError: Cannot read property 'crop' of undefined

### ISSUE THREAD

**Issue by tuhlmann** [JakeMakesStuff/MagicCap#230]
## Information

MagicCap Version: 2.1.2
System OS: Linux 5.8.0-7642-generic / Platform: linux
Installation ID: dc7dd822-aa7f-41f1-8f13-a5496e5f3756
Config: {"hotkey":null,"upload_capture":true,"uploader_type":"magiccap","clipboard_action":2,"save_capture":true,"save_path":"/home/tuhlmann/Bilder/MagicCap/","light_theme":false,"install_id":"dc7dd822-aa7f-41f1-8f13-a5496e5f3756"}

### Description 
When I try to create a screenshot of a Gif capture, I see the following error reported:

```
TypeError: Cannot read property 'crop' of undefined
    at /ewu/MagicCap/resources/app/node_modules/screenshot-desktop/lib/linux/index.js:167:69
```

### Steps to reproduce
Looking at existing errors of the screenshot-desktop package, I think it's related to having 2 connected monitors.
All I did was install the application by building master from source and by downloading the latest release. I did not change any settings.

When I unplug my external monitor it takes a screenshot.

Both of my monitors use fractional scaling on X11.

**Comment by sniff122** [comment]
yeah ive noticed that magiccap doesnt work correctly with multiple displays, i have a feeling its how displays IDs are named, on my laptop i have the eDP build in display and my HDMI port is HDMI-A-0. It kinda looks like the project is dead considering that v3 was apparently in the works but hasnt been released yet, even after idk how many months
my issue (#206) also covers this issue
